src/components/jsonviewer/jsonviewer.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, h, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'json-viewer',
+  styleUrl: 'jsonviewer.scss',
+  shadow: true,
+})
+export class JSONViewer {
+  @Prop() data: any
+
+  render() {
+    return (
+      <div>
+        {'{'}
```

Comment:
I wish there was a nicer way to put a curly brace in my component, but using a js string of it is the best bet right now.